Automated Smoketest: :x:
OS: centos 7.2.1511 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: c7f0b9f6b36556b684e4201892cd996e36b9dbc1
Number of Error(s): 6
/mnt/disk1/home/vamosax/smoketest/PR-11115/HPCC-Platform/esp/services/ws_esdlconfig/ws_esdlconfigservice.hpp:74:41: error: no matching function for call to \xe2\x80\x98SpinBlock::SpinBlock(const SpinLock&)\xe2\x80\x99

[Build error:](https://gist.github.com/5ca3f9a5529949c4249bbf37ce829344)
Build log [Build log](https://gist.github.com/HPCCSmoketest/5ca3f9a5529949c4249bbf37ce829344#file-result-18-05-03-23-26-43-log)

--- a/esp/services/ws_esdlconfig/ws_esdlconfigservice.hpp
+++ b/esp/services/ws_esdlconfig/ws_esdlconfigservice.hpp
@@ -67,7 +67,8 @@ public:
         return true;
     }
 private:
-    bool m_isDetachedFromDali;
+    //Service will be attached to Dali until told otherwise
+    bool m_isDetachedFromDali = false;
 };
 
 #endif //_ESPWIZ_WsESDLConfig_HPP__

